From 78d58deb5afbe4fc06319553d4ed975053589d00 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 20 Oct 2017 15:29:07 +0200
Subject: network: Spawn wifi panel for further WPA enterprise configuration

Settings recently split Wi-Fi configuration from the Network panel,
so launch that instead.

https://bugzilla.gnome.org/show_bug.cgi?id=789231
---
 js/ui/status/network.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index 02a43c1..9575af7 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -951,7 +951,7 @@ var NMWirelessDialog = new Lang.Class({
                 || (accessPoints[0]._secType == NMAccessPointSecurity.WPA_ENT)) {
                 // 802.1x-enabled APs require further configuration, so they're
                 // handled in gnome-control-center
-                Util.spawn(['gnome-control-center', 'network', 'connect-8021x-wifi',
+                Util.spawn(['gnome-control-center', 'wifi', 'connect-8021x-wifi',
                             this._device.get_path(), accessPoints[0].dbus_path]);
             } else {
                 let connection = new NetworkManager.Connection();

